tsconfig.json

```diff
@@ -46,7 +46,8 @@
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
     "typeRoots": [
       "node_modules/@types",
-      "types"
+      "types",
+      "typings"
     ] /* List of folders to include type definitions from. */,
     // "types": [],                           /* Type declaration files to be included in compilation. */
```

Comment:
Should this rather be completely removed?

Reply:
Yes, I think it should. The types folder was there by default and didn't check for redundancy when adding the new one.